webplugin/js/app/km-rich-text-event-handler.js

```diff
@@ -305,14 +306,10 @@ Kommunicate.richMsgEventHandler = {
         } else {
             form.submit();
         }
-        if(replyText){
-            var messagePxy = {
-                'message': replyText, //message to send 
-            };
-    
-            Kommunicate.sendMessage(messagePxy);
-        }
-
+        var messagePxy = {};
```

Comment:
@jithinbalu is it going to be set only for dialogflow case or for all whoever is using form?

Reply:
@devashishmamgain  KM_CHAT_CONTEXT  will be added in the metadata if they did not set formActionUrl in the dialogflow JSON and messagePxy.message will be set if replyText is present (for all users)

Reply:
@jithinbalu @Suraj-Kumar currently we are assuming that if it is not URL then it is dialogflow chat context, can we have a different identifier for dialogflow chat context? in future we might have to build one for amazon lex and other platforms

Reply:
@devashishmamgain @Suraj-Kumar "postBackToBotPlatform" identifier is added for bot platform chat context and
requestType can have 3 values
empty - submit a form on the action url
json - make a xhr request on action url
postBackToBotPlatform - send data back to bot platform.